libs/internal/include/launchdarkly/data_model/sdk_data_set.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <boost/json/value.hpp>
+#include <launchdarkly/data_model/item_descriptor.hpp>
+#include <launchdarkly/data_model/segment.hpp>
+#include <optional>
+#include <tl/expected.hpp>
+#include <unordered_map>
+
+namespace launchdarkly::data_model {
+
+struct SDKDataSet {
+    using FlagKey = std::string;
+    using SegmentKey = std::string;
+    // std::unordered_map<FlagKey, ItemDescriptor<Flag>> flags;
```

Comment:
Commented out code.

Reply:
(This one is added in the next (flag model) PR)